09/calculator.go

```diff
@@ -0,0 +1,13 @@
+package calculator
+
+func Subtract(x int, y int) int {
+	return x - y
+}
+
+func Multiply(x int, y int) int {
+	return x * y
+}
+
+func Divide(x int, y int) int {
+	return (x / y)
```

Comment:
We can omit the parentheses here.

```suggestion
	return x / y
```